Makefile

```diff
@@ -102,13 +102,14 @@ check-xgo:
 
 cross-compile-server: cross-compile-server-build-loop
 	tree dist
+	tree /build
 	@$(foreach platform,$(subst ;, ,$(PLATFORMS)),TARGETOS=$(firstword $(subst |, ,$(platform))) TARGETARCH=$(word 2,$(subst |, ,$(platform))) make normalize-server-artifacts || exit 1;)
 
 cross-compile-server-build-loop:
 	$(foreach platform,$(subst ;, ,$(PLATFORMS)),TARGETOS=$(firstword $(subst |, ,$(platform))) TARGETARCH=$(word 2,$(subst |, ,$(platform))) make release-server-xgo || exit 1;)
 
 normalize-server-artifacts:
-	mv dist/server/$(TARGETOS)/$(TARGETARCH)/$(shell ls dist/server/$(TARGETOS)/$(TARGETARCH)/) dist/server/$(TARGETOS)/$(TARGETARCH)/woodpecker-server
+	mv /build/woodpecker-server-$(TARGETOS)-$(TARGETARCH) dist/server/$(TARGETOS)/$(TARGETARCH)/woodpecker-server
```

Comment:
Hi, thanks fpr trying my suggestions.

I didn't try it with the `PLATFORMS` specified in `.woodpecker/docker.yml`, which is why I didn't run into the same problem. The problem is, that xgo produces binaries, where the `TARGETARCH` is not exactly the same as specified. Especially annoying is `arm64/v8` becomes `arm64` in the output binary, whereas `arm/v7` becomes `arm-7`.

The patch below should fix that, although it feels kind of hacky
```patch
-       mv /build/woodpecker-server-$(TARGETOS)-$(TARGETARCH) dist/server/$(TARGETOS)/$(TARGETARCH)/woodpecker-server
+       mv /build/woodpecker-server-$(TARGETOS)-$(subst arm64/v8,arm64,$(subst arm/v,arm-,$(TARGETARCH))) dist/server/$(TARGETOS)/$(TARGETARCH)/woodpecker-server
```